app/main/appium.js

```diff
@@ -321,9 +321,9 @@ function connectClientMethodListener () {
 }
 
 const getCurrentSessions = _.debounce(async (evt, data) => {
-  const {host, port, ssl} = data;
+  const {host, port, path, ssl} = data;
   try {
-    const res = await request(`http${ssl ? 's' : ''}://${host}:${port}/wd/hub/sessions`);
+    const res = await request(`http${ssl ? 's' : ''}://${host}:${port}${path ? path : '/wd/hub'}/sessions`);
```

Comment:
You can just set 'wd/hub' as the default above

`const {host, port, path:appiumPath = 'wd/hub', ssl} = data;`

And then this is just:

`http${ssl ? 's' : ''}://${host}:${port}${appiumPath}/sessions`